assets/js/app/LocusZoom.js

```diff
@@ -151,5 +210,13 @@
         return response.promise;
     };
 
+    exports.isString = function(x) {
```

Comment:
I've found it very helpful to have a set of consistent isString, isFunction, isArray methods. I think your suggested edit is good. It would be even better to keep those as private functions and not expose them outside the module.